models/iosxr/bgp_global/iosxr_bgp_global.yaml

```diff
@@ -0,0 +1,826 @@
+module: iosxr_bgp_global
+short_description: Manages BGP global resource module.
+description:
+- This module configures and manages the attributes of BGP global on Cisco IOS-XR platforms.
+version_added: 1.0.0
```

Comment:
This should be the next minor release version of IOS-XR.

Reply:
Ack, made changes